index.ts

```diff
@@ -0,0 +1,186 @@
+// https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md#run-lighthouse-multiple-times
+import { spawnSync } from 'child_process';
+import { computeMedianRun } from 'lighthouse/lighthouse-core/lib/median-run.js';
+import psi from 'psi';
+// https://stackoverflow.com/a/62499498/9931154
+import { createRequire } from 'module';
+import { type Options } from './options.js';
+const metaRequire = createRequire(import.meta.url);
+const lighthouseCli = metaRequire.resolve('lighthouse/lighthouse-cli');
+import { v4 as uuid } from 'uuid';
+import fs from 'fs';
+import chalk from 'chalk';
+
+const NUM_RUNS = 5;
+const platforms = ['mobile', 'desktop'] as const;
+
+const colorMap = {
```

Comment:
`chalk` is awesome! Thanks for adding this!!

Reply:
It's inspiring! I'll be using it for all of my cli personal projects going forward!